bdbstore/src/test/java/org/apache/qpid/server/store/berkeleydb/replication/ReplicatedEnvironmentFacadeTest.java

```diff
@@ -1196,6 +1202,36 @@ public void onNoMajority()
                    masterListener.awaitForStateChange(State.MASTER, _timeout, TimeUnit.SECONDS));
     }
 
+    public void testCommit() throws Exception
+    {
+        ReplicatedEnvironmentFacade master = createMaster();
+        assertEquals("Unexpected message store durability",
+                     new Durability(Durability.SyncPolicy.NO_SYNC, Durability.SyncPolicy.NO_SYNC, Durability.ReplicaAckPolicy.SIMPLE_MAJORITY),
+                     master.getRealMessageStoreDurability());
+        assertEquals("Unexpected durability", TEST_DURABILITY, master.getMessageStoreDurability());
+        assertTrue("Unexpected coalescing sync", master.isCoalescingSync());
+
+        doNothing().when(_tx).commit();
+        doNothing().when(_coalescingCommiter).commit(Mockito.any(), Mockito.anyBoolean());
+        master.commit(_tx, false);
```

Comment:
This test does not really a test commit functionality.

Reply:
Remove the test